api/src/controllers/cards.controller.ts

```diff
@@ -30,15 +30,36 @@ export class CardsController {
     try {
       // Initialize our page to 1 and our limit to 12 (12 cards per page)
       let page: number = Number(req.query.page) || 1;
+      if(page < 0) {
+        page = 1; 
+      }
```

Comment:
I think this should be if (page < 1) because if page=0 is entered then the page will still be 0 instead of 1.

Reply:
You're right, nice catch